examples/satellite-azure/variables.tf

```diff
@@ -190,3 +198,103 @@ variable "host_labels" {
     error_message = "Label must be of the form `key:value`."
   }
 }
+
+##################################################
+# IBM CLOUD ROKS Cluster Variables
+##################################################
+
+variable "create_cluster" {
+  description = "Create Cluster: Disable this, not to provision cluster"
+  type        = bool
+  default     = true
```

Comment:
Do we really want to default this to `true`?

Reply:
yeah, i still think this should default to `false`. I don't think we should be changing the default behavior this drastically in the example in this PR.